### DISCUSSION TITLE
can we customise NotToggle label to (not allowed or any custom label like notTrue)

### DISCUSSION THREAD

**Discussion by kuljeet2604** [react-querybuilder/react-querybuilder#620] [Q&A] [answered]
can we customise NotToggle label to (not allowed or any custom label like notTrue)
currently we have "Not" by default

**Comment by jakeboone02** [comment, marked as answer]
Use the [`translations` prop](https://react-querybuilder.js.org/docs/components/querybuilder#translations), like this:

```
<QueryBuilder translations={{ notToggle: { label: "Not true" } }} />
```

**Reply by chinku04** [reply]
worked
